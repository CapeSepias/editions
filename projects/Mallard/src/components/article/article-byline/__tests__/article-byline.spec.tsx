import React from 'react'
import TestRenderer, { ReactTestRendererJSON } from 'react-test-renderer'
import { ArticleByline } from '..'

describe('ArticleByline', () => {
    it('should display a default Byline', () => {
        const component: ReactTestRendererJSON | null = TestRenderer.create(
            <ArticleByline>Gary Younge</ArticleByline>,
        ).toJSON()
        expect(component).toMatchSnapshot()
    })
})
