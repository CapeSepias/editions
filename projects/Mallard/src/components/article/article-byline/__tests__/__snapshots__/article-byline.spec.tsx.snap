// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleByline should display a default Byline 1`] = `
<Text
  style={
    Array [
      Object {
        "flexShrink": 0,
        "fontFamily": "GuardianTextEgyptian-Bold",
        "fontSize": 18,
        "lineHeight": 22,
      },
      Array [
        Object {
          "paddingBottom": 10,
        },
        Object {
          "color": "#121212",
        },
        undefined,
      ],
    ]
  }
>
  Gary Younge
</Text>
`;
